<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingVerticalReverse</class>
 <widget class="QDialog" name="SettingVerticalReverse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting VerticalReverse</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>113</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Strength</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxTopSide">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>260</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Top side</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxStrength">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>110</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>140</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>170</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:0-0.5   Normally 0.16</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>290</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>J:/検査画面icon/flash_64.PNG</normaloff>J:/検査画面icon/flash_64.PNG</iconset>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxWidth">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>140</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-0.500000000000000</double>
   </property>
   <property name="maximum">
    <double>0.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.160000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>290</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>J:/検査画面icon/flash_swf_64.PNG</normaloff>J:/検査画面icon/flash_swf_64.PNG</iconset>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxBottomSide">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bottom side</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbLayer2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbLayer0">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer 0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbLayer1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer 1</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxGamma">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>217</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pixel Gamma</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
